Src/FluentAssertions/AssertionExtensions.Async.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions.Specialized;
+
+namespace FluentAssertions
+{
+    public static partial class AssertionExtensions
```

Comment:
I would keep it with the rest of the extension methods. No need for partial classes here.